apps/nrm/views/pv-dob.html

```diff
@@ -0,0 +1,8 @@
+{{<partials-page}}
+  {{$page-content}}
+    {{#fields}}
+      {{#renderField}}{{/renderField}}
+    {{/fields}}
+    {{#input-submit}}continue{{/input-submit}}
+  {{/page-content}}
+{{/partials-page}}
```

Comment:
Hof should just render a normal page, do you need this page? Can you delete and check?